client/components/main/Desktop/TypeSelector/index.js

```diff
@@ -72,12 +93,28 @@ const RequestTypeSelector = ({
                       padding: '0 0 0 9px',
                     }}
                     checked={selectedTypes[type.typeId]}
+                    
                     onChange={() => updateTypesFilter(type.typeId)}
                   />
                 }
                 label={type.typeName}
               /> 
             ))}
+            <FormControlLabel 
+              key={'all'}
```

Comment:
I don't think you'll need to wrap the key value in curly braces if it's just a string.